structuredevent-model/src/main/java/com/sequenceiq/cloudbreak/structuredevent/event/cdp/CDPStructuredEventContainer.java

```diff
@@ -0,0 +1,52 @@
+package com.sequenceiq.cloudbreak.structuredevent.event.cdp;
+
+import java.util.Collections;
+import java.util.List;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class CDPStructuredEventContainer {
+
+    private List<CDPStructuredFlowEvent> flow;
+
+    private List<CDPStructuredRestCallEvent> rest;
+
+    private List<CDPStructuredNotificationEvent> notification;
+
+    public CDPStructuredEventContainer() {
+    }
+
+    public CDPStructuredEventContainer(
+        List<CDPStructuredFlowEvent> flow,
+        List<CDPStructuredRestCallEvent> rest,
+        List<CDPStructuredNotificationEvent> notification) {
+        this.flow = flow;
+        this.rest = rest;
+        this.notification = notification;
+    }
+
+    public List<CDPStructuredFlowEvent> getFlow() {
+        return flow != null ? flow : Collections.emptyList();
```

Comment:
why not initialize the field with a `Collections.emptyList()` and then the getter won't have any logic. At least I think we usually handle this way. same applies to the others